library/ws-widget/collection/src/lib/_common/user-leaderboard/user-leaderboard.component.html

```diff
@@ -89,7 +95,10 @@
               </ws-widget-avatar-photo>
             </div>
           <div class="rank-user-name">{{rank3?.fullname}}</div>
-          <div class="rank-user-points">{{rank3?.total_points ? rank3?.total_points : 0}}</div>
+          <div class="rank-user-points flex">
+            <img class="podium-karmapoints-icon" alt="karma points" src="/assets/icons/home/karma-badge.svg">
+            {{rank3?.total_points ? rank3?.total_points : 0}}
```

Comment:
Make sure, this syntax working fine

Reply:
Itis working, I tested with the rank3 user data.